cmd/daemon/contract_test.go

```diff
@@ -13,15 +14,18 @@ import (
 )
 
 func TestGetOrDeploySwapFactory(t *testing.T) {
-	pk, err := ethcrypto.HexToECDSA(common.DefaultPrivKeyXMRTaker)
+	pk, err := ethcrypto.HexToECDSA(tests.GetTakerTestKey(t))
 	require.NoError(t, err)
 
 	ec, err := ethclient.Dial(common.DefaultEthEndpoint)
 	require.NoError(t, err)
+	defer ec.Close()
+
+	tmpDir := t.TempDir()
```

Comment:
The `TempDir()` function from `testing.T` creates a 2nd-level temporary directory in `/tmp` and removes it when the test is complete. The name includs the test name if you are ever debugging.